src/main/scala/ai/privado/utility/Utilities.scala

```diff
@@ -437,4 +437,105 @@ object Utilities {
     })
     semanticList.toList
   }
+
+    def reportUnresolvedMethods(xtocpg: Try[Cpg], filename: String): Unit = {
+    var total = 0
+    var unresolvedSignatures = 0
+    var unresolvedNamespaces = 0
+    var unresolvedSignaturesList = ListBuffer[String]()
+    var unresolvedNamespacesList = ListBuffer[String]()
+
+    val java_unresolved_signature = "(?i)(.*)(unresolved)(signature)(.*)"
+    val java_unresolved_namespace = "(?i)(.*)(unresolved)(namespace)(.*)"
+    val python_unresolved = "(?i)(.*)(unknownfullname)(.*)"
```

Comment:
@gaurav-gogia Confirm With @hiteshbedre and @dbMundada about what else to consider here. Would recommend including `ANY`  also

Reply:
No need to have `Any` in pattern.